idaes/generic_models/unit_models/pressure_changer.py

```diff
@@ -455,6 +479,38 @@ def actual_work(b, t):
                     b.work_isentropic[t] * b.efficiency_isentropic[t]
                 )
 
+        # Add head expressions (often calcuated from performance curve)
```

Comment:
This comments doesn't seem to match what follows - it mentions performance curves, but they don't appear in the following `try` statement.